docker/layers.bzl

```diff
@@ -71,11 +89,51 @@ def assemble(ctx, layers, tags_to_names, output, stamp=False):
       mnemonic = "JoinLayers"
   )
 
-def incremental_load(ctx, layers, images, output, stamp=False):
+def incremental_load(ctx, images, output, stamp=False):
   """Generate the incremental load statement."""
   stamp_files = []
   if stamp:
     stamp_files = [ctx.info_file, ctx.version_file]
+
+  load_statements = []
```

Comment:
+1 on rewriting the incr_load script in python. Python generating bash makes me sad.